articles/quickstart/backend/java-spring-security/01-authorization.md

```diff
@@ -91,20 +87,15 @@ public class SecurityConfig extends WebSecurityConfigurerAdapter {
 
 ## Configure the Protected Endpoints
 
-The example below shows how to implement CRUD (create, read, update and delete) methods and a login route for the Photos resource. 
-
-For CRUD methods, you need authentication and specific scopes. 
+The example below shows how to implement API methods.
 
 In the `AppConfig` class, add route matchers to the snippet. The `hasAuthority()` method provides a way to specify the required scope for the resource.
 
-The routes shown below are available for the following requests: 
+The routes shown below are available for the following requests:
 
-- `GET /login`: available for non-authenticated requests
-- `GET /photos`: available for authenticated requests containing an access token with the `read:photos` scope granted
-- `POST /photos`: available for authenticated requests containing an access token with the `create:photos` scope granted
-- `PUT /photos`: available for authenticated requests containing an access token with the `update:photos` scope granted
-- `DELETE /photos`: available for authenticated requests containing an access token with the `delete:photos` scope granted
-- Any other route that doesn't match the above requires the user to be authenticated with no additional scopes
+- `GET /api/public`: available for non-authenticated requests
```

Comment:
This lines are not platform specific and are shared across multiple API quickstarts. Please move them to a snippet file